catalog/spatial/geocoding/spatial-geocoding-lucene/src/main/java/org/codice/ddf/spatial/geocoding/query/GeoNamesQueryCatalog.java

```diff
@@ -0,0 +1,300 @@
+/**
```

Comment:
This now queries for all gazetteer data in the metacard core right?  If so, I think we should rename this

Reply:
Renamed